content-src/news_updates.yaml

```diff
@@ -8,25 +8,6 @@
 #   dev: false
 #   content: >
 #     Example content goes here
--
-  slug: general-update-3
```

Comment:
It's always been the plan to remove news updates from the YAML. [The slug (which should be unique) is a part of the l10n IDs](https://github.com/mozilla/testpilot/blob/master/frontend/tasks/util.js#L63)